msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/b2c/B2CIdLabSisoPolicyTest.java

```diff
@@ -46,7 +46,6 @@
 
 import java.util.Arrays;
 
-@RunWith(Parameterized.class)
```

Comment:
When There is no test now, using this class we just can keep this annotation as it is.
Though I understand if no test is using it, what's the point of having this annotation and also if this annotation requires any processing where some cost is spent in terms of resources/time. Just a thought